app/models/katello/product_content.rb

```diff
@@ -46,8 +46,8 @@ def self.content_table_name
 
     def self.enabled(organization)
       content_ids = Katello::RootRepository.in_organization(organization).where.not(content_id: nil).pluck(:content_id)
-      structured_apt_content_ids = Katello::Repository.in_organization(organization).library.pluck(:content_id)
-      joins(:content).where("#{self.content_table_name}.cp_content_id" => content_ids + structured_apt_content_ids)
+      deb_content_ids = Katello::Repository.in_organization(organization).library.pluck(:content_id)
```

Comment:
```suggestion
      deb_content_ids = Katello::Repository.in_organization(organization).library.deb_type.pluck(:content_id)
```
I noticed this query could be made a bit more efficient by only looking at deb-type repos.

Reply:
Applied.